HelloWeb/README.md

```diff
@@ -0,0 +1,29 @@
+# HelloWeb Application
+
+Gluon Application on the browser
+
+## Basic Requirements
+
+A list of the basic requirements can be found online in the [Gluon documentation](https://docs.gluonhq.com/#_requirements).
+
+## Quick instructions
+
+### Run the sample
+
+    mvn gluonfx:run
+
+### Run the sample in browser
+
+    mvn gluonfx:build gluonfx:nativerun -Pweb
+
+NOTE: Gluon currently supports Chrome, Chromium, and Firefox browser.
+
+### Desktop
```

Comment:
Is this relevant here?

Reply:
All our other samples work natively on Desktop, so does this sample.

Reply:
Yes, but I doubt this is relevant.
When reading this, it seems like these are the next steps to deploy on web. It is not clear that this has nothing to do with running on the web.
In this light, this sample is different from the other samples we have.

Reply:
In that case, I will remove it and also the related files from `META-INF`